src/commands/dev/dev-exec.mjs

```diff
@@ -16,7 +16,8 @@ const devExec = async (cmd, options, command) => {
     env = await getEnvelopeEnv({ api, context: options.context, env, siteInfo })
   }
 
-  await injectEnvVariables({ devConfig: { ...config.dev }, env, site })
+  env = await addDotEnvVariables({ devConfig: { ...config.dev }, env, site })
```

Comment:
nit: This method is using the same pattern as `getEnvelopeEnv` (i.e. receive `env` and return an augmented version of it), and yet one is `getX` and the other is `addX`. It'd be nice to make them consistent.

Reply:
Done